teos/src/tls.rs

```diff
@@ -0,0 +1,94 @@
+/* The following code for generating mTLS certificates is taken from:
+ * https://github.com/ElementsProject/lightning/blob/master/plugins/grpc-plugin/src/tls.rs
+ *
+ * This file is licensed under the BSD-MIT license, as described here:
+ * https://github.com/ElementsProject/lightning/blob/master/LICENSE
+*/
+
+use anyhow::{Context, Result};
+use rcgen::{Certificate, KeyPair};
+use std::path::Path;
+
+// Just a wrapper around a certificate and an associated keypair.
```

Comment:
This should be `///` instead of `//` so it generates docs. Applies to all docs in this file.